File: algorithms/1_easy/C/02980_check_if_bitwise_or_has_trailing_zeros/solution.cpp
class Solution {
public:
    bool hasTrailingZeros(vector<int>& nums) {
        int even_count {0};

        for (const int& num : nums) {
            if ((num & 1) == 0) {
                even_count += 1;

                if (even_count > 1) {
                    return true;
                }
            }
        }

        return false;
    }
};

File: algorithms/1_easy/C/02980_check_if_bitwise_or_has_trailing_zeros/solution.rs
impl Solution {
    pub fn has_trailing_zeros(nums: Vec<i32>) -> bool {
        let mut even_count = 0_i32;

        for &num in nums.iter() {
            if (num & 1) == 0 {
                even_count += 1;

                if even_count > 1 {
                    return true;
                }
            }
        }

        false
    }
}
